Grid/Source/Entity.php

```diff
@@ -245,6 +245,10 @@ protected function getFieldName($column, $withAlias = false)
 
                 return "$functionWithParameters as $alias";
             }
+            // To handle query from filter with necessary HAVING clause which does not accept alias
+            if (!$withAlias && $hasHaving) {
```

Comment:
The variable $hasHaving comes from nowhere.

I guess that it should be $hasHavingClause given as parameter of the function ?